core/client/pom.xml

```diff
@@ -188,6 +188,12 @@
           <scope>provided</scope>
         </dependency>
         <dependency>
+          <groupId>com.google.inject</groupId>
+          <artifactId>guice</artifactId>
+          <version>3.0</version>
```

Comment:
Since Alluxio does not directly dependent those indirect dependencies, when the version of those dependencies bumps in the third-party library, the version we set in our repo could be stale. This could lead to broken dependencies because that third-party library may depend on some API only available in the newer version.